source/views/streaming/streams/list.js

```diff
@@ -55,6 +56,9 @@ export class StreamListView extends React.PureComponent {
 
   getData = async () => {
     try {
+      const start = Date.now()
+      this.setState(() => ({refreshing: true}))
```

Comment:
Either this shouldn't default to `false`, or it shouldn't be called here, because this function is called as soon as the component is loaded, so refreshing effectively starts out set to true.

Reply:
What will set refreshing to `true`, then?

Reply:
Actually, scratch this. If you do what I recommended in my fourth comment, this is irrelevant and correct.